plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/utils/CodeTransformUtils.kt

```diff
@@ -43,6 +44,15 @@ fun refreshToken(project: Project) {
     provider.refresh()
 }
 
+fun getQTokenProvider(project: Project) = (
```

Comment:
Question: Not a blocker. Could maybe be in a more shared folder across teams in the core package?